src/main/java/org/veupathdb/service/eda/compute/plugins/betadiv/BetaDivPlugin.java

```diff
@@ -0,0 +1,58 @@
+package org.veupathdb.service.eda.compute.plugins.betadiv;
+
+import org.jetbrains.annotations.NotNull;
+import org.veupathdb.service.eda.common.client.spec.StreamSpec;
+import org.veupathdb.service.eda.common.model.VariableDef;
+import org.veupathdb.service.eda.common.plugin.util.PluginUtil;
+import org.veupathdb.service.eda.compute.plugins.AbstractPlugin;
+import org.veupathdb.service.eda.compute.plugins.PluginContext;
+import org.veupathdb.service.eda.compute.RServe;
+import org.veupathdb.service.eda.generated.model.BetaDivPluginConfig;
+import org.veupathdb.service.eda.generated.model.BetaDivPluginRequest;
+
+
+import java.util.List;
+
+public class BetaDivPlugin extends AbstractPlugin<BetaDivPluginRequest, BetaDivPluginConfig> {
+
+  private static final String INPUT_DATA = "beta_div_input";
+
+  public BetaDivPlugin(@NotNull PluginContext<BetaDivPluginRequest, BetaDivPluginConfig> context) {
+    super(context);
+  }
+
+  @NotNull
+  @Override
+  public List<StreamSpec> getStreamSpecs() {
+    return List.of(new StreamSpec(INPUT_DATA, getConfig().getCollectionVariable().getEntityId())
```

Comment:
Is there a place in the compute service framework to validate the plugin request?  Thinking we should be checking if the input var is a collection var and maybe its type (number, integer, date?) if that matters.

Reply:
Nothing I saw yet, but I didn't look very hard tbh.